deploy/crds/eunomia_v1alpha1_gitopsconfig_crd.yaml

```diff
@@ -56,6 +56,7 @@ spec:
               enum:
               - Retain
               - Delete
+              - None
```

Comment:
Hmmm ... I think this file is auto generated by the `operator-sdk`, so I think the "correct" way to update this might be to update the Go Lang code then run an `operator-sdk` command to regenerate the file.

But I could be wrong.

Reply:
@etsauer you cannot edit a generated file, you need to add the option to the annotation in the `gitopsconfig_types.go` file line: 52. then regenerate everything with:
`operator-sdk generate k8s`
`operator-sdk generate openapi`

Reply:
@raffaelespazzoli yep did that. See latest commit with the updated `gitopsconfig_types.go`, but generation isn't making the changes I expect.